<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondGameDialog</class>
 <widget class="QDialog" name="SecondGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="answerComboBox"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="artistLabel">
     <property name="text">
      <string>Художник</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="exitPushButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGraphicsView" name="pictureGraphicsView"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Описание</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="nextPushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ваш ответ</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Как называется эта картина?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
